### PR TITLE
Release 1.62.1

### PR DESCRIPTION
**Pull request by enejb**
Release for Gutenberg Mobile 1.62.1

## Related PRs
Gutenberg: https://github.com/WordPress/gutenberg/pull/35123
WPAndroid  https://github.com/wordpress-mobile/WordPress-Android/pull/15371
WPiOS      https://github.com/wordpress-mobile/WordPress-iOS/pull/17216

## Extra PRs that Landed After the Release Was Cut
https://github.com/WordPress/gutenberg/pull/34957
https://github.com/WordPress/gutenberg/pull/35061

### Change 1
- **PR:** https://github.com/WordPress/gutenberg/pull/34957 
- **Issue:** Image block - Fix height and border regression

### Change 2
- **PR:** https://github.com/WordPress/gutenberg/pull/35061
- **Issue:** Column: fix width float attribute cut off 

## Test plan

- Use the main WP apps to test the changes above.
- Smoke test the main WP apps for [general writing flow](https://github.com/wordpress-mobile/test-cases/tree/master/test-cases/gutenberg/writing-flow).
- Test the regressions have been fixed. 
See 
https://github.com/WordPress/gutenberg/pull/35061 and 
https://github.com/WordPress/gutenberg/pull/34957

## Release Submission Checklist

- [ ] Verify Items from test plan have been completed
- [ ] Approve and run optional Android and iOS UI tests
- [ ] Check if `RELEASE-NOTES.txt` is updated with all the changes that made it to the release. Replace `Unreleased` section with the release version and create a new `Unreleased` section.
- [ ] Check if `gutenberg/packages/react-native-editor/CHANGELOG.md` is updated with all the changes that made it to the release. Replace `## Unreleased` with the release version and create a new `## Unreleased`.
- [ ] Bundle package of the release is updated.

### CODE REVIEW

**Comment by enejb** [comment]
### Writing Flow checklist
Device: Pixel 5, Android 11
Build: WordPress-pr-15371-build-115724.apk - from https://github.com/wordpress-mobile/WordPress-Android/pull/15371

#### General
- [x] TC001 - Paste formatted text copied from website
- [x] TC001 - Multiline components
#### Rich Text Format
- [x] TC001 - Bold, Italic, strikethrough buttons
- [x] TC002 - Alignment buttons
- [x] TC003 - Alignment Split
- [x] TC004 - Link button works without selection
- [x] TC005 - Link button works with a selected word
- [x] TC006 - Adding a link from a copied URL
- [x] TC007 - Test format detection under the cursor
- [x] TC008 - Test formatting doesn't remove leading or trailing whitespace
##### Splitting and merging
- [x] TC001 - Merge after writing
- [x] TC002 - Merge after selection
- [x] TC003 - Merge after deleting text
- [x] TC004 - Merge after deleting all
- [x] TC005 - Merge multiple blocks
- [x] TC006 - Splitting/merge list block
#### Undo / Redo - Test Cases
- [x] TC001 - Undo/redo block actions
- [x] TC002 - Undo/redo text
- [x] TC003 - Undo/redo text format

**Comment by enejb** [comment]
## Test Bug fixes

Device: Pixel 5, Android 11
Build: WordPress-pr-15371-build-115724.apk - from wordpress-mobile/WordPress-Android#15371

- [x] PR: [Mobile] - Image block - Fix height and border regression WordPress/gutenberg#34957
- [x] PR: [Mobile] - Column: fix width float attribute cut off WordPress/gutenberg#35061

**Comment by jd-alexander** [comment]
### Writing Flow checklist

Device: iPhone 11
Build: 18.3 from https://github.com/wordpress-mobile/WordPress-iOS/pull/17216

#### General
- [x] TC001 - Paste formatted text copied from website
- [x] TC001 - Multiline components
#### Rich Text Format
- [x] TC001 - Bold, Italic, strikethrough buttons
- [x] TC002 - Alignment buttons
- [x] TC003 - Alignment Split
- [x] TC004 - Link button works without selection
- [x] TC005 - Link button works with a selected word
- [x] TC006 - Adding a link from a copied URL
- [x] TC007 - Test format detection under the cursor
- [x] TC008 - Test formatting doesn't remove leading or trailing whitespace
##### Splitting and merging
- [x] TC001 - Merge after writing
- [x] TC002 - Merge after selection
- [x] TC003 - Merge after deleting text
- [x] TC004 - Merge after deleting all
- [x] TC005 - Merge multiple blocks
- [x] TC006 - Splitting/merge list block
#### Undo / Redo - Test Cases
- [x] TC001 - Undo/redo block actions
- [x] TC002 - Undo/redo text
- [x] TC003 - Undo/redo text format

**Comment by jd-alexander** [comment]
## Test Bug fixes

Device: iPhone 11
Build: 18.3 from https://github.com/wordpress-mobile/WordPress-iOS/pull/17216

- [x] PR: [Mobile] - Image block - Fix height and border regression WordPress/gutenberg#34957
- [x] PR: [Mobile] - Column: fix width float attribute cut off WordPress/gutenberg#35061

File: RELEASE-NOTES.txt [hunk 1]
```diff
@@ -1,6 +1,11 @@
 Unreleased
 ---
 
+1.62.1
+------
+* [**] Image block: fix height and border regression. [https://github.com/WordPress/gutenberg/pull/34957]
+* [**] Column block: fix width attribute flout cutoff. [https://github.com/wordpress-mobile/gutenberg-mobile/pull/3921]
```
Comment (fluiddot) [inline RELEASE-NOTES.txt:7]: @enejb I noticed that this entry is referencing a different PR and found a typo:
- `Column block: fix width attribute flout cutoff.` => `Column block: fix width float attribute cut off.`
- `https://github.com/wordpress-mobile/gutenberg-mobile/pull/3921` => `https://github.com/WordPress/gutenberg/pull/35061`

Comment (fluiddot) [reply RELEASE-NOTES.txt:7]: Heads up that I'm addressing this issue in https://github.com/WordPress/gutenberg/pull/35432.

**Pull request merged**